.changeset/little-drinks-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@strapi/pack-up': major
+---
+
+line in the sand
```

Comment:
what is this file? 😆

Reply:
they're changesets! https://github.com/changesets/changesets

Reply:
Oh, nice, looks good!